things/api/things/http/endpoint_test.go

```diff
@@ -1192,15 +1192,16 @@ func TestUpdateChannel(t *testing.T) {
 	ts := newServer(svc)
 	defer ts.Close()
 
-	schs, _ := svc.CreateChannels(context.Background(), token, channel)
-	sch := schs[0]
+	chs, err := svc.CreateChannels(context.Background(), token, channel)
+	require.Nil(t, err, fmt.Sprintf("unexpected error: %s\n", err))
+	ch := chs[0]
 
-	ch := channel
-	ch.Name = "updated_channel"
-	updateData := toJSON(ch)
+	c := channel
+	c.Name = "updated_channel"
+	updateData := toJSON(c)
```

Comment:
just curios, why this and not `[]byte("updated_channel")`

Reply:
Didn't really go into test designs. I was mostly just renaming and refactoring a style.